bot/README.md

```diff
@@ -24,13 +24,13 @@ If those tests are OK, you can run the bot locally, by specifying a Taskcluster
 ```
 export TRY_TASK_ID=XXX
 export TRY_TASK_GROUP_ID=XXX
-code-review-bot --taskcluster-secret=path/to/secret
+code-review-bot --configuration=path/to/config.yaml
```

Comment:
Mention it above too `by specifying a Taskcluster secret with your configuration` -> `by specifying a Taskcluster secret or a local YAML file with your configuration`